server/Entities/ScheduledPlaceToEatRepository.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Linq;
+using Entities.Interfaces;
+using Entities.Models;
+
+namespace Entities
+{
+  public class ScheduledPlaceToEatRepository: IScheduledPlaceToEatRepository
+  {
+    private readonly IRepositoryContext RepositoryContext;
+
+    public ScheduledPlaceToEatRepository(IRepositoryContext repositoryContext)
+    {
+      RepositoryContext = repositoryContext;
+    }
+
+    public IQueryable<ScheduledPlaceToEat> GetScheduledPlaceToEatByUserId(int UserId)
+    {
+      return RepositoryContext.ScheduledPlacesToEat.Where(u => u.UserId.Equals(UserId));
+    }
+
+    public ScheduledPlaceToEat GetScheduledPlaceToEatById(int Id)
+    {
+      return RepositoryContext.ScheduledPlacesToEat.FirstOrDefault(u => u.Id.Equals(Id));
+    }
+
+    public int CreateScheduledPlaceToEat(ScheduledPlaceToEat scheduledPlaceToEat)
+    {
+      RepositoryContext.ScheduledPlacesToEat.Add(scheduledPlaceToEat);
+      RepositoryContext.SaveChanges();
+      return scheduledPlaceToEat.Id;
```

Comment:
SaveChanged - mutable function? it is not pretty good

Reply:
what if saving was unsuccess?

Reply:
there will be an exception that will be processed in the service and the server will continue working. null will be returned

Reply:
what should I use instead of SaveChanges?